public/less/_repository.less

```diff
@@ -1607,8 +1607,8 @@
 	}
 }
 
-.commit-list {
-	vertical-align: baseline;
+tbody.commit-list {
+    vertical-align: baseline;
```

Comment:
Please fix tab back to what is in rest file

Reply:
But, that would contradict [`.editorconfig`](https://github.com/go-gitea/gitea/blob/35cc5b0402d46d672e02bbe1ad15d1460077e8f4/.editorconfig#L18-L20). See https://github.com/go-gitea/gitea/pull/3064 for fixing the `.less` files indendation issues.

Reply:
Also, the upper half of that file is already intended using spaces, btw :)